src/traits.rs

```diff
@@ -274,6 +274,65 @@ pub trait Random: Sized {
     fn random(rng: &mut impl CryptoRngCore) -> Self;
 }
 
+/// Possible errors of [`RandomBits::try_random_bits`].
+#[cfg(feature = "rand_core")]
+#[derive(Debug)]
+pub enum RandomBitsError {
```

Comment:
For completeness, this could probably use a `Display` impl along with a `std::error::Error` impl when the `std` feature is enabled

Reply:
Done